Calculator.c

```diff
@@ -0,0 +1,111 @@
+#include<stdio.h>
+#include<stdlib.h>
+#include<string.h>
+#include<ctype.h>
+#define MAX 80
```

Comment:
Practice naming variables and function names such that they are self-descriptive and meaningful. This applies throughout the code, mentioning here only once.